deploy/kubernetes/helm/che/custom-charts/che-keycloak/values.yaml

```diff
@@ -15,3 +15,4 @@ image: quay.io/eclipse/che-keycloak:nightly
 requireAdminPasswordChange: true
 keycloakAdminUserName: admin
 keycloakAdminUserPassword: admin
+deploymentStrategy: RollingUpdate
```

Comment:
> Allow to set the deployment strategy, the default RollingUpdate fail if multiattach is not supported by volumes

I think in most cases it won't be supported. What about setting default to recreate which mean - introduce an ability to override default but not change the behavior we already have?

Reply:
yes

Reply:
but other charts postgres, che, already have Recreate as deploy strategy